vnext/PropertySheets/JSEngine.props

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Import Project="$(SolutionDir)\ExperimentalFeatures.props" Condition="Exists('$(SolutionDir)\ExperimentalFeatures.props')" />
   <PropertyGroup>
+    <JsEnginePropsDefined>true</JsEnginePropsDefined>
```

Comment:
Should this be at the bottom and only true if Hermes or V8 are enabled?

Reply:
@dannyvv My understanding was the the property sheets of a given project is evaluated in a single thread serially .. Is that true ? 

And this property is supposed to represent any JS engine properties  .. Not just V8/Hermes ..